StudyHub/StudyHub.BLL/Services/Auth/AuthService.cs

```diff
@@ -91,16 +81,15 @@ public async Task<AuthSuccessDTO> RefreshTokenAsync(string accessToken, string r
         if (DateTimeOffset.UtcNow > user.RefreshToken.ExpiryDate)
             throw new ExpiredException("Refresh token is expired");
 
-        if (user.RefreshToken.Token != refreshToken)
+        if (user.RefreshToken.Token != request.RefreshToken)
             throw new IncorrectParametersException("Refresh token is invalid");
 
-        return await GetAuthTokens(user);
+        return await GetAuthTokensAsync(user);
     }
 
-    private async Task<AuthSuccessDTO> GetAuthTokens(User user)
+    protected async Task<AuthSuccessDTO> GetAuthTokensAsync(User user)
     {
-        var roles = (await _userManager.GetRolesAsync(user)).ToArray();
-        return new AuthSuccessDTO(_tokenService.GenerateJwtToken(user!, roles),
+        return new AuthSuccessDTO(await _tokenService.GenerateJwtTokenAsync(user!),
```

Comment:
I dont get why delete roles from there?

Reply:
We don't need roles declaration to be found that high away from usage